ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -4,4 +4,5 @@
     <filter root="/content/hclecomm" mode="merge"/>
     <filter root="/content/dam/hclecomm" mode="merge"/>
     <filter root="/content/experience-fragments/hclecomm" mode="merge"/>
+	<filter root="/oak:index/hclecommLuceneIndex" />
```

Comment:
@rakeshkumar-verma : add mode merge here

Reply:
@rakeshkumar-verma : why do we have <filter root="/oak:index/hclecommLuceneIndex" /> as part of ui.content pom ? why not add in apps ?

Reply:
@rishabh-behl  merge mode updated in filter. Build was breaking while including oak:index in ui.apps due to root filter root's ancestor.